express.js

```diff
@@ -275,9 +279,9 @@ app.get(`/${process.env.CALLBACK}`, passport.authenticate('faforever', {
   failureRedirect: '/login', // Failed auth
   failureFlash: true
 }), function (req, res) {
-  res.redirect('/');
-  //res.redirect(fullUrl ? fullUrl : '/');
-  //fullUrl = '/'; // Successful auth
+  //Success Auth
+  res.redirect(previousURL ? previousURL : '/');
```

Comment:
this will now work with multiple users, previousURL is a global variable (used by all users)